### PR TITLE
[chore] Add ROSA Linux worker for e2e

### PR DESCRIPTION
**Pull request by KraMorK**
## Description

Add ROSA Linux worker for e2e
## Why do we need it, and what problem does it solve?

Adding RosaOS for e2e tests

## Why do we need it in the patch release (if we do)?

Not related to release

## What is the expected result?

E2e tests use RosaOS

## Checklist
- [x] e2e tests passed.

## Changelog entries
Add ROSA Linux worker for e2e

```changes
section: testing
type: chore
summary: Add ROSA Linux worker for e2e.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/static'...

  :fast_forward: `e2e: Static, Containerd, Kubernetes 1.27` for `deckhouse:chore/add-rose-for-static-test` [started](https://github.com/deckhouse/deckhouse/actions/runs/10792809310).

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/static'...

**Comment by KraMorK** [comment]
/e2e/abort static;Static;containerd;1.27 10808600465 10808600465-1-con-1-27 /sys/deckhouse-oss/install:pr9681 astra@192.168.199.133

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/static'...

**Comment by KraMorK** [comment]
/e2e/abort static;Static;containerd;1.27 10809713705 10809713705-1-con-1-27 /sys/deckhouse-oss/install:pr9681 astra@192.168.199.207

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/static'...

**Comment by KraMorK** [comment]
/e2e/abort static;Static;containerd;1.27 10810581046 10810581046-1-con-1-27 /sys/deckhouse-oss/install:pr9681 astra@192.168.199.38

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/static'...

File: testing/cloud_layouts/Static/rosa-instance-bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2023 Flant JSC
```
Comment (borg-z) [inline testing/cloud_layouts/Static/rosa-instance-bootstrap.sh:3]: 2024

**Comment by alexey-gavrilov-flant** [comment]
/e2e/abort static;Static;containerd;1.27 10792624495 10792624495-1-con-1-27 /sys/deckhouse-oss/install:pr9681 astra@192.168.199.118

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @alexey-gavrilov-flant. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by alexey-gavrilov-flant** [comment]
/e2e/abort static;Static;containerd;1.27 10809713705 10809713705-1-con-1-27 /sys/deckhouse-oss/install:pr9681

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @alexey-gavrilov-flant. I've recognized your '/e2e/abort' command and started the workflow...

File: testing/cloud_layouts/Static/infra.tpl.tf [hunk 2]
```diff
@@ -22,6 +22,7 @@ terraform {
   required_providers {
     openstack = {
       source = "terraform-provider-openstack/openstack"
+      version = "2.1.0"
```
Comment (KraMorK) [inline testing/cloud_layouts/Static/infra.tpl.tf:25]: пока фиксируем, обновлю потом, куча брейкинг ченжей
https://github.com/terraform-provider-openstack/terraform-provider-openstack/blob/main/CHANGELOG.md#300--25-september-2024-

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/static'...

**Comment by name212** [comment]
/e2e/abort static;Static;containerd;1.27 11260798253 11260798253-1-con-1-27 /sys/deckhouse-oss/install:pr9681 astra@192.168.199.235

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/static'...

**Comment by borg-z** [comment]
e2e/abort static;Static;containerd;1.27 11269460788 11269460788-1-con-1-27 /sys/deckhouse-oss/install:pr9681 astra@192.168.199.65

**Comment by borg-z** [comment]
/e2e/abort static;Static;containerd;1.27 11269460788 11269460788-1-con-1-27 /sys/deckhouse-oss/install:pr9681 astra@192.168.199.65

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/static'...

**Review by name212 — APPROVED** [review]

**Pull request merged**